src/components/Header/elements.js

```diff
@@ -134,7 +93,40 @@ export const Overlay = styled.div`
   }
 `
 
-export const DesktopMenu = styled(Flex)`
+const DesktopMenuList = styled.ul`
+  display: flex;
+  flex-direction: column;
+  padding: ${remcalc(24)};
+  padding-right: ${remcalc(0)};
+  padding-top: ${remcalc(12)};
+  justify-content: center;
+
+  ${breakpoint('phone')`
+    padding-bottom: ${remcalc(24)};
+  `}
+
+  ${breakpoint('tablet')`
+    width: auto;
+    height: auto;
+    flex-direction: row;
+    position: relative;
+
+    > li:not(:last-child) {
+      margin-right: ${remcalc(18)};
+    }
+  `};
+
+  @media screen and (max-width: 768px) and (min-width: 600px) {
+    padding: ${remcalc(36)};
+    a {
+      top: ${remcalc(0)};
+    }
+  }
+`
+
+const DesktopMenuContainer = styled(Flex).attrs({
```

Comment:
We suggestion would be to...

1. Perhaps rename
- `DesktopMenuContainer` -> `Navbar`
- `DesktopMenuList` -> either `UnorderedList` (if these styles apply to any/all lists in our website) or `NavbarList` (if these styles only apply to an unordered list in the navbar context)?

2. Move Navbar & UnorderedList/NavbarList into a separate file called `Navbar.js`

3. Rename `elements.js` as `HamburgerMenu.js` or `MobileNav.js` (or something else?), with the understanding that this file is the one to be sorted as part of [the mobile nav ticket](https://trello.com/c/d1sAHhFE/389-13-dropdown-component-mobile)

4. Finally, we could move the `DesktopMenu` code directly into `Header/index.js`?

I feel like it's worth explicitly having this inside the render method, since it's the bit that controls whether the mobile hamburger menu is open or not:
```
<DesktopMenuContainer open={open}>
  <DesktopMenuList>{children}</DesktopMenuList>
</DesktopMenuContainer>
```

or rather, if you agree with the new naming, this code would exist in `Header/index.js`:

```
<Navbar open={open}>
  <NavbarList>{children}</NavbarList>
</Navbar>
```

Reply:
I honestly don't think this is the time (sprint)/place (this PR) to make these changes. If you're writing them down to come back to them later, sure, otherwise I don't see the need

Reply:
Fair enough if you don't think this is the place to do the refactor and move things around :+1: Cos this whole navbar work has been deprioritised now.

But I reckon the renaming is worth doing now, since it's slightly misleading?